pack/chore.go

```diff
@@ -88,9 +89,20 @@ func (chore *Chore) Run(ctx context.Context) {
 		chore.loop = sync2.NewCycle(chore.interval)
 
 		go func() {
```

Comment:
Nothing is waiting for this goroutine to be completed. https://www.storj.io/blog/production-concurrency explains this problem further.

So there's a chance that it ends up using a closed `db.DB` or `*Store`.

Reply:
We could try to do it over a google meet tomorrow.